drivers/interrupt_controller/intc_plic.c

```diff
@@ -22,12 +22,15 @@
 #include <zephyr/irq.h>
 
 #define PLIC_BASE_ADDR(n) DT_INST_REG_ADDR(n)
+#define PLIC_REG_PRIO_OFFSET 0x0
```

Comment:
should this be `0x1000`?

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/a2b238eee77cf9137db858618eb2881c53963330/dts/riscv/virt.dtsi#L168-L171

prio offset should be 0, 0x1000 is the offset of pending bits according to the specs